htmx-spring-boot/src/main/java/io/github/wimdeblauwe/htmx/spring/boot/mvc/HxReswap.java

```diff
@@ -0,0 +1,25 @@
+package io.github.wimdeblauwe.htmx.spring.boot.mvc;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation to specify how the response will be swapped.
+ * See <a href="https://htmx.org/attributes/hx-swap/">hx-swap</a> for possible values.
+ *
+ * @see <a href="https://htmx.org/reference/#response_headers">HX-Reswap</a>
+ */
+@Target({ElementType.TYPE, ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface HxReswap {
+
+    /**
+     * A value to specify how the response will be swapped.
+     *
+     * @see <a href="https://htmx.org/attributes/hx-swap/">hx-swap</a>
+     */
+    String value();
```

Comment:
Should we return an enum value here to make it easier to select the correct values?

Reply:
The value of `HX-Reswap` supports the same options as the [hx-swap](https://htmx.org/attributes/hx-swap/) attribute. So, in my opinion, the current implementation that only supports the values of the `HxSwapType` enum in `HtmxResponse` is wrong. I would fix it by adding a class that also supports all modifier options. Let me just do that for a second.